hooks/tk-multi-publish2/basic/publish_session.py

```diff
@@ -12,7 +12,7 @@
 import pprint
 import sgtk
 
-from pyfbsdk import FBApplication
+from pyfbsdk import *
```

Comment:
Importing `*` is discouraged. I suggest importing what you need explicitly:

`from pyfbsdk import FBApplication, FBFilePopup, FBFilePopupStyle`